src/index.html

```diff
@@ -1,16 +1,54 @@
-<!doctype html>
+  <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+        name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+        data-qa="big"
+      >
+      <label class="searchLabel">
+        <img
+        src="images/Search.svg"
+        alt="search"
+        class="bigImage"
```

Comment:
```suggestion
        <img
          src="images/Search.svg"
          alt="search"
          class="bigImage"
```
pay attention to the code style, fix it all over the page

Reply:
How can I remove the default border?